apis/v1alpha1/installationTypesHelper.go

```diff
@@ -21,5 +17,5 @@ func isMultitenant(installType InstallationType) bool {
 }
 
 func IsManaged(installType InstallationType) bool {
-	return installType == InstallationTypeManaged || installType == InstallationTypeManagedApi || isMultitenant(installType)
+	return installType == InstallationTypeManagedApi || isMultitenant(installType)
```

Comment:
I think we should make a clear difference between what managed is and what multitenant is. 
This function does make much sense to me anymore since managed in this scenario can be managed-api or multitenant-manged-api, since this is all what intly operator can be, how can this return false?

Reply:
After doing a bit more digging into this function and where it is used. The function its self is only used once in the preflight checks when checking for the Pager Duty secret. I am in agreement now, this function can be fully removed.